datafusion/common/src/error.rs

```diff
@@ -450,6 +399,69 @@ impl DataFusionError {
         #[cfg(not(feature = "backtrace"))]
         "".to_owned()
     }
+
+    fn error_name(&self) -> &'static str {
```

Comment:
nit: This feels more like an `error_prefix` to me than a name?

Reply:
I agree